posix.mak

```diff
@@ -383,7 +383,8 @@ clean :
 	rm -rf $(ROOT_OF_THEM_ALL) $(ZIPFILE) $(DOC_OUTPUT_DIR)
 
 zip :
-	zip $(ZIPFILE) $(MAKEFILE) $(ALL_D_FILES) $(ALL_C_FILES) index.d win32.mak win64.mak osmodel.mak
+	-rm -f $(ZIPFILE)
+	zip -r $(ZIPFILE) . -x .git\* -x generated\*
```

Comment:
Walter is the only one using the zip targets, so naturally they are not maintained.
Requiring everyone to maintain a list they are not using and regularly complaining that they don't doesn't make sense.